src/Middleware/FormatterMiddleware.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace CultuurNet\CalendarSummaryV3\Middleware;
+
+use Closure;
+use CultuurNet\SearchV3\ValueObjects\Offer;
+
+interface FormatterMiddleware
+{
+    public function format(Offer $offer, Closure $next): string;
```

Comment:
@jdrieghe what's the main reasoning behind using the middleware? Because in the current implementation I don't seem to see an added value.

Reply:
Ah, I had a similar question but our posts crossed as I was typing them as part of a review.

Reply:
Right now I implemented the "stack" of middleware very naively because we only had one. If another middleware is added, we can define them as an array and traverse the middleware chain by passing them through each other.

Reply:
Sounds good to me!